utils/tracerAPI/api.ts

```diff
@@ -197,6 +203,78 @@ export const fetchTradeStats: (params: {
     return fetchedTradeStats;
 };
 
+export const fetchNextPoolState: (params: { network: KnownNetwork; pool: string }) => Promise<NextPoolState> = async ({
+    network,
+    pool,
+}) => {
+    const route = `${TRACER_API}/poolsv2/nextPoolState?network=${network ?? NETWORKS.ARBITRUM}&poolAddress=${pool}`;
+
+    const fetchedNextPoolState: NextPoolState = await fetch(route)
+        .then((res) => res.json())
+        .then((nextPoolState: NextPoolStateAPIResponse) => {
+            return {
+                currentSkew: new BigNumber(nextPoolState.currentSkew),
```

Comment:
Not a dealbreaker by any means, but curious as to why we are using bignumber.js instead of the ethers BigNumber implementation.